packages/encode/src/rlp-flow.js

```diff
@@ -0,0 +1,237 @@
+/**
+ * Built on top of rlp library, removing the BN dependency for the flow.
```

Comment:
Any chance you can add a link to the original here. Also include its license please.

Reply:
Fixed the issue, added their full license block inside for more clarity.
Check and resolve.